programs/ssl/ssl_server2.c

```diff
@@ -2962,10 +3001,13 @@ int main(int argc, char *argv[])
         } else
 #endif /* MBEDTLS_HAVE_TIME */
         {
-            if ((ret = mbedtls_ssl_ticket_setup(&ticket_ctx,
-                                                rng_get, &rng,
-                                                opt.ticket_aead,
-                                                opt.ticket_timeout)) != 0) {
+            if ((ret = mbedtls_ssl_ticket_setup(&ticket_ctx, //context
+                                                rng_get, //f_rng
+                                                &rng, //p_rng
+                                                opt.ticket_alg, //alg
+                                                opt.ticket_key_type, //key_type
+                                                opt.ticket_key_size, //key_type
```

Comment:
Typo in comment. Also, I'm not sure if the comments are that helpful here, considering the names of the variables are already quite explicit.